build.cake

```diff
@@ -1,7 +1,7 @@
-#addin "nuget:?package=Cake.FileHelpers"
-#addin "nuget:?package=Cake.Git"
-#addin "nuget:?package=Cake.Incubator"
-#tool "nuget:?package=GitVersion.CommandLine"
+#addin "nuget:?package=Cake.FileHelpers&version=3.1.0"
+#addin "nuget:?package=Cake.Git&version=0.19.0"
+#addin "nuget:?package=Cake.Incubator&version=3.1.0"
+#tool "nuget:?package=GitVersion.CommandLine&version=4.0.0"
```

Comment:
These are all the latest stable versions, except for Git.Incubator which is one version back because version 4.0.0 had breaking changes.